docs/src/overview/get-support.md

```diff
@@ -0,0 +1,40 @@
+---
+title: " Get support"
+weight: 4
+toc: false
+description: |
+  Find out how to get help If you're experiencing an issue with Platform.sh.
+---
+
+{{< description >}}
+
+
+## Create a support ticket
```

Comment:
As with the items below, this should include a description of when it should be chosen. Not all issues should be support tickets. For example, support doesn't really cover issues with app development. And what they do cover and how fast might be influenced by the plan. I don't think you have to get into all those details, but at least something would be helpful.

Reply:
I've added a description as per M.Antikaapo instructions.